pandas/io/parquet.py

```diff
@@ -116,13 +117,32 @@ def write(
                 **kwargs,
             )
 
-    def read(self, path, columns=None, **kwargs):
+    def read(self, path, columns=None, use_nullable_dtypes=False, **kwargs):
         path, _, _, should_close = get_filepath_or_buffer(path)
 
         kwargs["use_pandas_metadata"] = True
-        result = self.api.parquet.read_table(
-            path, columns=columns, **kwargs
-        ).to_pandas()
+        to_pandas_kwargs = {}
+        if use_nullable_dtypes:
+            if LooseVersion(self.api.__version__) > "0.15.1.dev":
```

Comment:
Should this raise for older versions?

Reply:
That's probably more informative yes, will add

Reply:
we automatically convert for period / interval right?

Reply:
do we not already require this version?